profiling/datadog-profiling.c

```diff
@@ -61,6 +63,12 @@ static void diagnose_profiling_enabled(bool enabled) {
   if (enabled) {
     string = "[Datadog Profiling] Profiling is enabled.";
   } else {
+    /* This message might not be logged. I've see-sawed back-and-forth on
+     * whether *_ENABLED=off + *_LOG_LEVEL=info should actually do anything. On
+     * one hand, the profiler is off and shouldn't do anything. On the other,
+     * if the log level is set to something, reminding people that it's set to
+     * off may be useful.
+     */
     string = "[Datadog Profiling] Profiling is disabled.";
```

Comment:
For those wondering, this currently does not log anything.

Reply:
For info, it probably should be silent, but there probably should be a level that emits "Profiling is disabled by {source of truth}" where source of truth should be env/ini/whatever ...

For later though ...

Reply:
+1